docs/modules/ROOT/pages/upgrade_to_version_2.adoc

```diff
@@ -0,0 +1,166 @@
+= Upgrade to Version 2.x
+:doctype: book
+
+== Configuration File Update
+
+In version 2.x:
+
+ - `RSpec/InvalidPredicateMatcher` cop is removed
+ - `CustomIncludeMethods` configuration option for `RSpec/EmptyExampleGroup` is removed
+ - cop departments are nested for cops with a department that doesn't match the extension name (`Capybara`, `FactoryBot`, `Rails`)
+
+[discrete]
+=== Adjust the configuration of `RSpec/EmptyExampleGroup`
+
+[source,yaml]
+----
+# .rubocop.yml
+
+# Before
+RSpec/EmptyExampleGroup:
+  CustomIncludeMethods:
+  - include_tests
+
+# After
+AllCops:
+  RSpec:
+    Language:
+      Includes:
+        Example:
+        - include_tests
+----
+
+=== Add a top-level `RSpec` department
+
+RuboCop extensions had cops with clashing names and departments, e.g. both `rspec-rails` and `rubocop-rspec` had `Rails::HttpStatus` cops.
+To avoid issues, e.g. inability to disable just one of the cops, each extension now has its own uber-department.
+Expectedly, RuboCop RSpec's uber-department name is `RSpec`.
```

Comment:
```suggestion
Expectedly, RuboCop RSpec’s uber-department name is `RSpec`.
```

Reply:
I'm still not sure if "uber-department" is a correct term, but couldn't think of a better one :D